docs/AddNewOp.md

```diff
@@ -1,58 +1,58 @@
-# Adding a New Operator to ONNX standard
-
-Operator is the basic computation unit in the neural network models.
-Currently, ONNX covers a core set of operators. There are operators which are
-needed by users, however they are still excluded from ONNX operator set. For
-example, GroupNormalization was proposed this year, and many developers already
-use it in their models, but in ONNX operator set, it's still missing. To make
-ONNX more comprehensive, we should encourage community to contribute high
-quality operator specification to expand our operator set.
-
-We are maintaining [a list](https://github.com/onnx/onnx/issues/1646) in our
-GitHub issue. Please check it and post your request following this thread.
-
-## Step 1: Determine Whether We Should Add the Operator
-Determine whether if the proposed operator is common enough, usually it
-should/will be supported by at least two frameworks, such as PyTorch/Caffe2,
-Tensorflow, MxNet, etc.
-
-## Step 2: Determine the Operator Should Be Added as Operator or Function
-If the requested operator can be easily expressed as several (e.g., 2 or 3)
-existing operators, we can define it as a Function (i.e., composite operator).
-MeanVarianceNormalization is the first example of registering an operator
-as a Function.
-
-## Step 3: Define and polish the spec of the proposed operator.
-The spec should include:
-1. opset version of the added operator. Check out our
+# Proposing and submitting a new operator or function to ONNX
+
+Operators are the basic building blocks that define ONNX model. With a rich set of operators, ONNX can describe most DNN and ML models from various frameworks. Functions allow for composing complex operators from more primitive operators. The ONNX specification includes a core set of operators that enable many models. It is a non-goal to add all possible operators, however more operators are added as needed to cover evolving needs.
+
+In this document, we describe the process of accepting a new proposed operator and how to properly submit a new operator as part of ONNX standard. The goal is to improve on what we currently have based on our experience, learning and feedbacks we gathered from the community.
+
+# 4 steps to add an operator
+1. Decide what to propose
+2. Submit PR for new operator/function
+3. Review of PR by Operators SIG
+4. Merging of PR and inclusion in next ONNX release
+
+# Step 1: Proposing a new operator/function
+In order to propose a new operator/function, the following is needed:
+1. If the operator can be composed by other ONNX operators, then it should be a function and not an operator (we have a function in ONNX : MeanVarianceNormalization).
```

Comment:
Based on my experience of implementing BatchNorm using this approach, the current framework lacks to full support. i.e. converting attributes to tensors etc. I think we need to make sure that we add the needed features to make this option suitable, we may even need additional operations to allow function operations to manipulate the tensors correctly

Reply:
@hobei can you propose what is missing?